shell/assets/translations/zh-hans.yaml

```diff
@@ -1102,7 +1112,7 @@ cluster:
     advanced:
       label: 高级选项
       detail: 对节点注册方式的额外控制。每个注册的节点需要的值，通常都不相同。
-      nodeName: 节点名称
+      NodeName: 节点名称
```

Comment:
Hi @bisht-richa , do you know why `Nodename` is used as the key here? It uses an uppercase initial.

Maybe the en-us.yaml file needs an update and needs to see where it's referenced.